blocks/reader-revenue-manager/block-editor-plugin/editor-styles.scss

```diff
@@ -16,7 +16,14 @@
  * limitations under the License.
  */
 
-/*
- * Block editor styles.
- */
-@import "components/reader-revenue-manager/googlesitekit-rrm-block-editor";
+.editor-sidebar {
```

Comment:
It looks like the `editor-sidebar` class is a [recent addition](https://github.com/WordPress/gutenberg/pull/60815/files#diff-de4b93789d100c3f45f34a01926c40c39206d34cce26efc3eb217e58becc91e3) and using it won't be backward compatible to our supported range of WP versions (from 5.4+ for this particular feature as per the AC, rather than 5.2 which is our minimum version for the plugin in general).

Please use a class which will be backward compatible to WP 5.4; if you are referencing a class that is provided by Gutenberg then it should be backward compatible to Gutenberg 6.6.0 as per https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/.

Reply:
Thanks @techanvil , I think we can safely proceed with `.block-editor__container` because this class exists in WP 5.4 as well as in the latest version. Please let me know your thoughts.

Reply:
Looks good, thanks @ankitrox!